src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -53,15 +53,15 @@ const AnonymousChannelListScreen = ({route}) => {
     <>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+        <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Search` component now receives an additional prop `isAnon` set to `true` and a modified `onPress` function. Ensure that the `Search` component can handle this new prop correctly and that the `goToContactScreen` function can accept an object with `{from: ANONYMOUS}` as its argument.
```diff
- <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+ <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->